partner_noncommercial/views/menu.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <menuitem
+        id="menu_partner"
+        name="Partners"
+        parent="contacts.menu_contacts"
+        groups="base.group_user"
```

Comment:
Perhaps a `group_partner_manager` is needed here as in its parent?

Reply:
In this case might be better to not have a group at all, like on the other submenu's. Having base.group_user will not make a material difference in practice. But @lfreeke would you like to delete the groups= line?